packages/@haiku/sdk-inkstone/src/index.ts

```diff
@@ -325,6 +325,14 @@ export namespace inkstone {
     export interface Organization {
       Name: string;
       PlanExpirationDate?: number;
+      Role: Role;
+      TrialStartDate?: number;
+      TrialDurationDays?: number;
+    }
+
+    export enum Role {
```

Comment:
https://github.com/HaikuTeam/mono/blob/60bf4679765206c63fa2f9462a060d58cecbed75/packages/%40haiku/sdk-inkstone/src/index.ts#L333-L336

nit: use `const enum` for better runtime perf ([check out how it compiles](https://www.typescriptlang.org/play/#src=enum%20EnumOnly%20%7B%0D%0A%20%20%20%20Free%20%3D%200%2C%0D%0A%20%20%20%20Pro%20%3D%202%2C%0D%0A%7D%0D%0A%0D%0Aconsole.log(EnumOnly.Free)%0D%0A%0D%0A%0D%0Aconst%20enum%20ConstEnum%20%7B%0D%0A%20%20%20%20Free%2C%0D%0A%20%20%20%20Pro%0D%0A%7D%0D%0A%0D%0Aconsole.log(ConstEnum.Free))) and since TS enums are number based, you don't have to specify the numbers! 

```suggestion
    export enum Role {
      Free,
      Pro,
    }
```

Reply:
On phone so haven't checked use case, but please note that if this is used outside sdk-inkstone, it can't be both const enum and usable.

Reply:
es, it is used outside, my bad! only improvement then could be to remove the explicit number declarations. Thanks stristr!

Reply:
Note that the association of the numbers to their symbols is very important here — this is "duplicated truth" from Inkstone (where an enum declares Free = 0 and Pro = 1).  If we remove the explicit number assignment here, we're one re-ordering (Pro, Free) away from this logic being very broken.  I'm in favor of explicit value assignment here as a safe-guard.

Reply:
that makes a lot of sense to me :)